docs/quickstart.rst

```diff
@@ -167,35 +205,44 @@ Mitigating disparity
 If we observe disparities between groups we may want to create a new model
 while specifying an appropriate fairness constraint. Note that the choice of
 fairness constraints is crucial for the resulting model, and varies based on
-application context. If selection rate is highly relevant for fairness in this
+application context. If balanced accuracy is highly relevant for fairness in this
 contrived example, we can attempt to mitigate the observed disparity using the
-corresponding fairness constraint called Demographic Parity. In real world
+corresponding fairness constraint called Equalized Odds. In real world
 applications we need to be mindful of the sociotechnical context when making
 such decisions. The Exponentiated Gradient mitigation technique used fits the
-provided classifier using Demographic Parity as the objective, leading to
-a vastly reduced difference in selection rate:
+provided classifier using Equalized Odds as the constraint and Error Rate
+as the objective, leading to a vastly reduced difference in  the selection rate:
 
 .. doctest:: quickstart
     :options:  +NORMALIZE_WHITESPACE
 
-    >>> from fairlearn.reductions import DemographicParity, ExponentiatedGradient
+    >>> from fairlearn.reductions import ErrorRateParity, EqualizedOdds, ExponentiatedGradient
     >>> np.random.seed(0)  # set seed for consistent results with ExponentiatedGradient
-    >>>
-    >>> constraint = DemographicParity()
+    >>> objective = ErrorRate(costs={'fp': 0.5, 'fn': 0.5})
```

Comment:
I'd love to have someone other than me cross-checking that this makes sense to them.

Reply:
Miro suggested `objective = ErrorRate(costs={'fp': 0.1, 'fn': 0.9})`, so let me know if that works for you and I can resolve this comment.